### PR TITLE
Update and rename CoatlAerospace.netkan to ProbesPlus.netkan

### PR DESCRIPTION
**Pull request by raveloda**
I looked at the guide and I hope I have the right changes to reflect the change of my mod to the new released version 0.14 . I'd appreciate if someone can help me get my mod updated, or just proofread my code chage. I suggest the name change to "ProbesPlus" because "Coatl Aerospace" is the name of the in-game company for my mods, whereas ProbesPlus is the mod name.

Also, the mod now has a Firespitter dependency. I also don't know what to put in the code to make "Bluedog Design Bureau" a suggested mod. I'll read the guide again.

### CODE REVIEW

File: NetKAN/ProbesPlus.netkan [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 	"$kref": "#/ckan/spacedock/768",
 	"license": "CC-BY-NC-SA-4.0",
-	"ksp_version": "1.1.2",
+	"ksp_version": "1.1.3",
 	"resources": {"homepage": "http://forum.kerbalspaceprogram.com/index.php?/topic/131145-wip-coatl-aerospace-probesplus-dev-thread-v013-beta-61316/",
 				"repository": "https://github.com/raveloda/Coatl-Aerospace"},
 	"depends": [{"name": "DMagicScienceAnimate"},
 		{"name": "DMagicOrbitalScience"},
-		{"name": "ModuleManager"}],
+		{"name": "ModuleManager"}{"name": "Firespitter"}],
```
Comment (politas) [inline NetKAN/ProbesPlus.netkan]: Needs a comma in between elements in braces {}

**Comment by politas** [comment]
The "name" (strictly "identifier", but relationship stanzas all say name for the mod identifier) for Bluedog Design Bureau is "BluedogDB".

**Comment by politas** [comment]
Also, can you please add the following section:

    "conflicts":[
        { "name": "CoatlAerospace" }
    ],

That should ensure people don't try to install this over the top of the old version.

**Comment by raveloda** [comment]
I think I made the correct changes. I will be honest, I do not understand what some of the changes mean. particularly how the "identifier" has no effect on the name. Where is the name defined?

**Comment by politas** [comment]
The name is drawn from Spacedock, along with author, abstract, version number and download location. It'll grab the ksp_version too if you delete that line.

**Comment by Olympic1** [comment]
@raveloda 
- Change `spec_version` back to `1` (this is the CKAN version, not mod version)
- Remove `ksp_version` as this will be pulled from Spacedock
- Line 16, change the `( )` with `{ }`
- Line 18, add a comma at the end
- Line 19, remove the comma at the end

**Comment by raveloda** [comment]
More changes. Thanks for proofreading it

**Comment by Olympic1** [comment]
Looks good, merging with my thanks

**Pull request merged**